src/actions/index.js

```diff
@@ -1,38 +1,115 @@
-let taskId = 0;
+import axios from "axios";
+import types from "src/actions/Types";
 
-export const addTask = (title, description) => ({
-  type: "ADD_TASK",
+export const getTasks = () => {
+  return async dispatch => {
+    try {
+      const res = await axios.get(`http://todo-backend-express.herokuapp.com/`);
+      if (res.status !== 200) {
```

Comment:
i dont remember but, this is necessary? the `try-catch` isnt enough?

Reply:
i'm reading [this](https://github.com/axios/axios) and it says that axios throws an error when the response code is out of the range of 2xx. So you are right 👍